modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -47,6 +51,13 @@ sub fetch_input {
 	my $release = $self->param_required('release');
 	my $division = $self->param('division');
 	my $metadata_script_options = "\$($meta_host details script) --release $release" . ($division ? " --division $division" : "");
+    my $rr_meta_db = $self->param('rr_meta_name');
+    my ($rr, $rr_meta_dba)     = _get_curr_rr_release($rr_meta_db, $meta_host);
```

Comment:
```suggestion
    my ($rr, $rr_meta_dba);
    ($rr, $rr_meta_dba) = _get_curr_rr_release($rr_meta_db, $meta_host) if $rr_meta_db;
```
(if you want to give the flexibility of not using rapid release genomes). If not, then the previous line should be `param_required()`.

Reply:
The perl linter complains now about that trailing `if $rrPmeta_db` - but I will put in the equivalent that Travis won't shout at me about.